cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/AbstractCgmesTapChangerBuilder.java

```diff
@@ -79,8 +79,8 @@ protected TapChanger build() {
         double value = point.asDouble(attr, defaultValue);
         if (Double.isNaN(value)) {
             context.fixed(
-                    "RatioTapChangerTablePoint " + attr + " for step " + step + " in table " + tableId,
-                    "invalid value " + point.get(attr));
+                "RatioTapChangerTablePoint " + attr + " for step " + step + " in table " + tableId,
+                () -> "invalid value " + point.get(attr));
```

Comment:
@lcosson I think there is no reason to create a Supplier here

Reply:
My bad, correcting this...